src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -96,6 +96,12 @@ class WorkspaceMembersPage extends React.Component {
             this.validate();
         }
 
+        if (prevProps.policyMemberList !== this.props.policyMemberList) {
```

Comment:
I'm pretty sure `policyMemberList` is an array, and `!==` can't be used to compare array length / values - they're just pointers in JS. So this is only checking that the pointers are the same, not that the values in the arrays are the same - is that what y'all are trying to do?

Reply:
Yes, that's exactly what we are trying to do. I have requested this change here https://github.com/Expensify/App/pull/18934#discussion_r1193846420.

Reply:
:sweat_smile: Yeah I'm sure. The idea is to do some logic only if the pointer change. If the pointer change then it's likely the values are changed too (if the pointer change but we have same values than this is a bad design somewhere else, yet it won't cause any bug if this ever happened). Also there is no case where the values change but keep using the same pointer (onyx do not mutate objects).  This is just an optimisation (comparing addresses should be the fastest).


Just curious though, how did you end up here :grin:?

Reply:
Innnnnnnnnteresting, that all sounds legit! 👍  Thanks for explaining :D

Hahaha I ended up here just because i saw some arrays being compared and had to ask why 😅